### PR TITLE
Added qos autodetect

### PR DESCRIPTION
**Pull request by gonzodepedro**
Signed-off-by: Gonzalo de Pedro <gonzalo@depedro.com.ar>

### CODE REVIEW

File: src/rqt_plot/rosplot.py [hunk 1]
```diff
@@ -101,12 +103,56 @@ def __init__(self, node, topic, start_time):
         topic_type, real_topic, fields = get_topic_type(node, topic)
         if topic_type is not None:
             self.field_evals = generate_field_evals(fields)
-            data_class = get_message(topic_type)
+            data_class = get_message_class(topic_type)
```
Comment (jacobperron) [inline src/rqt_plot/rosplot.py]: What's the reason for this change? IMO, it seems like a backwards step. I would prefer to use the utility further upstream, all else being equal.

**Review by jacobperron — COMMENTED** [review]
One comment, otherwise LGTM!

File: src/rqt_plot/rosplot.py [hunk 2]
```diff
@@ -38,8 +38,10 @@
 import threading
 import time
 
+from rclpy.qos import QoSDurabilityPolicy
 from rclpy.qos import QoSProfile
-from rosidl_runtime_py.utilities import get_message
+from rclpy.qos import QoSReliabilityPolicy
+from rqt_py_common.message_helpers import get_message_class
```
Comment (jacobperron) [inline src/rqt_plot/rosplot.py]: We need to also undo the change to the imports. We're using `get_message` from `rosidl_runtime_py` instead of `get_message_class`.

**Review by jacobperron — APPROVED** [review]

**Comment by gonzodepedro** [comment]
CI build: `--packages-up-to rqt_plot`
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/15041/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/9778/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/12731/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/15209/)
